### DISCUSSION TITLE
doris 现在支持事务了么？从什么版本开始支持的

### DISCUSSION THREAD

**Discussion by hcxx-x** [apache/doris#33327] [RFC]
再整合springboot和doris的时候发现可以使用@Transactional 来控制事务回滚了，不是说doris不支持事务的么？这里为什么可以回滚成功呢（doris版本 2.0.1）

**Comment by yagagagaga** [comment]
不支持，只是兼容了语法

**Reply by zengchen00** [reply]
不支持为什么又能回滚成功？
